src/vtok_agent/src/agent/mod.rs

```diff
@@ -194,6 +196,34 @@ impl PostSyncAction {
             });
     }
 
+    fn restart_nginx(wait_ms: u64) {
+        info!("Restarting NGINX.");
+
+        if let Ok(pid) = is_service_running("nginx.service") {
+            debug!("Sending SIGWINCH to PID={}", pid);
```

Comment:
Please add some info in the commit description about why `SIGWINCH` and `SIGQUIT` signals are used in these changes and what effect they have when sending to nginx.

Reply:
SIGQUIT was removed. Updated the last commit's description.